assets/app/view/game/game_info.rb

```diff
@@ -88,54 +60,51 @@ def phases
           end
 
           h(:tr, [
-            h(:td, td_props, phase[:name] + (current_phase == phase ? ' (Current) ' : '')),
-            h(:td, td_props, phase[:operating_rounds]),
-            h(:td, td_props, phase[:train_limit]),
+            h(:td, (current_phase == phase ? '⮕ ' : '') + phase[:name]),
```

Comment:
unicode looks inconsistent in browsers, what about -> or >

Reply:
It would be a crime to represent an arrow by the ASCII string `->` in the year 2020.

Reply:
lol ok maybe the arrow is ok or possible you can make an svg

Reply:
Arrows for consistency with spreadsheet (up/down arrow already indicates sort order there).
Even if it gets rendered slightly differently on different browsers it should still be clearly identifiable as an arrow ;) svg would just be more load.

Reply:
Real question: Is the arrow sufficient to indicate current round? Additionally highlight row with background color maybe?